_py1/solutions/base-sol.html

```diff
@@ -0,0 +1,5 @@
+---
+layout: default
+---
+
+<h1>Solution!</h1>
```

Comment:
Personally, I am not a fan of implementing solutions on another page - for example, you wouldn't be able to see the solutions for multiple questions on the same page. What are our thoughts on using Javascript to show-hide each solution?